docs/articles/documentation/test-api/built-in-waiting-mechanisms.md

```diff
@@ -76,7 +82,7 @@ test('My test', async t => {
 });
 ```
 
-## Waiting for Assertions to Pass
+## Wait Mechanism for Assertions
 
 TestCafe assertions feature the [Smart Assertion Query Mechanism](assertions/README.md#smart-assertion-query-mechanism).
```

Comment:
feature?

Reply:
Do you mean include? use?

Reply:
TestCafe assertions use the [Smart Assertion Query Mechanism](assertions/README.md#smart-assertion-query-mechanism) that is activated when you pass a [selector property](selecting-page-elements/selectors/using-selectors.md#obtain-element-state).